### ISSUE TITLE
Fullscreen bug on Linux Mint 19 Cinnamon

### ISSUE THREAD

**Issue by Fainec** [M7S/dockbarx#75]
Hello

When I get an application in fullscreen and the dockbarx is in dock behavior, the fullscreened window appears under the dock and not **over**

I got this message though when running dockx in console when a window is fullscreen.

`/usr/lib/python2.7/dist-packages/dockbarx/windowbutton.py:194: Warning: Source ID 3133 was not found when attempting to remove it
  gobject.source_remove(self.item.deopacify_sid)`

I installed the package from the unofficial ppa.
Any idea ?

**Comment by denisfa** [comment]
Out of curiosity, is this still a problem? If it is, can you provide some extra information? So it is possible to emulate your setup:

1. Get your configuration for dockbarx:
```
gconftool --dump /apps/dockbarx > dockbarx.xml
```
2. Are you using any special applets or other data? Can you send your `~/.dockbarx`?

3. What application are you trying to fullscreen with dockbar? Youtube on `chromium`? Some video player like `mpv`?

Put everything inside a zip and attach as a comment here.

**Comment by Fainec** [comment]
Hello Denisfa
Here are the files you asked me.

Otherwise, the problem still persist, and it appears when I fullscreen Youtube on Chrome, vivaldi, firefox, on VLC, MPV or smplayer and every others fullscreen app like games.

[dockbarx.zip](https://github.com/M7S/dockbarx/files/2645562/dockbarx.zip)

**Comment by denisfa** [comment]
Thanks for the quick reply! I am getting used to a couple of apps, and wanted to try cinnamon.

I had no luck, I only managed to get it working on Cinnamon when it crashed and started in Fallback Mode (took me some time to figure it out that it was the Fallback Mode that "fixed" the behaviour).
It has something to do with the WM, but it is a bit of mystery for me. I don't think it is a bug from the DE, more likely something that needs to be done slightly different.

I will play around this, see if I can learn a thing or two.

**Comment by Fainec** [comment]
Hello Denisfa,
Yes it surely a Cinnamon bug, because it appears only when I updated my Linux Mint 18 to Linux Mint 19 on the same Desktop Environnement but updated as well though.

I opened a ticket on the cinnamon github, they told me that they just fixed the problem on their side, but the patch is still not available on the realease branch of cinnamon.
https://github.com/linuxmint/Cinnamon/issues/7818#issuecomment-415602077

However, thank you for taking to help me.
Best regards
